components/script_bindings/codegen/CodegenRust.py

```diff
@@ -7126,7 +7150,7 @@ def manualImplClone(self):
     def manualImpl(self, t):
         if t == "Clone":
             return self.manualImplClone()
-        raise f"Don't know how to impl {t} for dicts."
+        raise ValueError(f"Don't know how to impl {t} for dicts.")
```

Comment:
Raising a `str` is invalid python and would result in 
```
TypeError: exceptions must derive from BaseException
```
at runtime.

My IDE was complaining about that.